src/kmrbase.c

```diff
@@ -97,6 +104,11 @@ kmr_init_2(int ignore)
     }
 #endif
 
+#if KMR_TRACE_ENABLE
```

Comment:
Calling these two functions in kmr_create_context is better, when the tracer is implemented as an option to kmr struct.